examples/nft-quest/nuxt.config.ts

```diff
@@ -14,29 +14,23 @@ export default defineNuxtConfig({
     "@nuxtjs/color-mode",
     "@nuxtjs/seo",
     "@vueuse/motion/nuxt",
+    "nuxt-gtag",
   ],
-  colorMode: {
```

Comment:
All of these changes in the `nuxt.config.ts` file are a result of re-ordering by linting. Also discovered `vite` config was duplicated in these files and removed the duplication.